examples/kubernetes/configure.sh

```diff
@@ -3,6 +3,13 @@
 # This script generates config.sh, which is a site-local config file that is not
 # checked into source control.
 
+# Erase any existing file since we append from here on.
+> config.sh
+
+# Custom Docker image.
+read -p "Vitess Docker image (leave empty for default) []: "
```

Comment:
We talked about this offline and agreed to keep this prompt for now. I'd like to drop in some kind of template system (i.e. something better than sed) soon to keep things maintainable when I add a third "environment". Currently there's "regular" and "benchmark" and I plan to add the option to use k8s "alpha/beta" features. We can think about the best UX for customization at that time.